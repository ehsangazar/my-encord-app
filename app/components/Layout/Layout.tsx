import Header from "../Header/Header";

interface LayoutProps {
  children: React.ReactNode;
}

const Layout = ({ children }: LayoutProps) => {
  return (
    <div className="flex h-screen bg-gray-50">
      {/* <Sidebar /> */}
      <div className="flex-1 h-full overflow-y-auto">
        <Header />
        <main className="h-full p-4 max-w-4xl mx-auto">{children}</main>
      </div>
    </div>
  );
};

export default Layout;
