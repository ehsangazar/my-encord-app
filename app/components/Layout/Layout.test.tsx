import renderer from "react-test-renderer";

import Layout from "./Layout";

it("Layout component should render correctly", () => {
  const tree = renderer.create(<Layout>Click me</Layout>).toJSON();
  expect(tree).toMatchSnapshot();
});
