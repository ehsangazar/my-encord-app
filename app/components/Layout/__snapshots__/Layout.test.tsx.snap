// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render correctly 1`] = `
<div
  className="flex h-screen bg-gray-50"
>
  <div
    className="flex-1 h-full overflow-y-auto"
  >
    <header
      className="flex items-center justify-between p-4 bg-gray-800 border-b"
    >
      <div
        className="max-w-4xl mx-auto w-full flex items-center justify-between"
      >
        <div
          className="flex items-center"
        >
          <h2
            className="text-xl font-semibold text-gray-800 dark:text-white"
          >
            My Encord App
          </h2>
        </div>
      </div>
    </header>
    <main
      className="h-full p-4 max-w-4xl mx-auto"
    >
      Click me
    </main>
  </div>
</div>
`;
